tests/conftest.py

```diff
@@ -255,7 +259,7 @@ def output_directory(tmp_path: Path) -> Path:
 def application_pid(
     in_container: bool, application_process: subprocess.Popen, application_docker_container: Container
 ) -> int:
-    pid = application_docker_container.attrs["State"]["Pid"] if in_container else application_process.pid
+    pid: int = application_docker_container.attrs["State"]["Pid"] if in_container else application_process.pid
```

Comment:
Why does it need `: int` here if we're assigning an `int` right away?

Reply:
`application_docker_container.attrs["State"]["Pid"]`, mypy can't be sure it's an `int`

Reply:
Hmm, I supposed that you need to use `cast` in that case